source/indexes/compound-index.txt

```diff
@@ -0,0 +1,99 @@
+.. _cpp-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Compound indexes** are indexes that hold references to multiple
+fields within a collection's documents. These indexes improve multi-field query and
+sort performance.
+
+To create a compound index, call the ``create_index()`` method and specify a document containing: 
+
+- The fields on which to create the index
+- The sort order for the indexed values, ``1`` for ascending or ``-1`` for descending.
```

Comment:
S: we also try to avoid [starting lists with articles](https://www.mongodb.com/docs/meta/style-guide/style/lists/#list-items)
```suggestion
- Fields on which to create the index
- Sort order for the indexed values, ``1`` for ascending or ``-1`` for descending.
```